source/math/FloatEuclidTransform.ooc

```diff
@@ -43,6 +46,22 @@ FloatEuclidTransform: cover {
 		this rotation == other rotation &&
 		this scaling == other scaling
 	}
+	convolveCenter: static func (euclidTransforms: VectorList<This>, kernel: FloatVectorList) -> This {
+		result := This new()
+		if (euclidTransforms count > 0) {
+			result scaling = 0.0f
+			quaternions := VectorList<Quaternion> new(euclidTransforms count)
+			for (i in 0 .. euclidTransforms count) {
+				result translation = result translation + euclidTransforms[i] translation * kernel[i]
```

Comment:
I agree though, can't you just make sure you access `euclidTransforms` and `kernel` once? I can fix it for you if you don't want to.